packages/react/src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -284,17 +310,49 @@ const TableToolbar = ({
         totalSelected={totalSelected}
         translateWithId={(...args) => tableTranslateWithId(i18n, ...args)}
       >
-        {batchActions.map(({ id, labelText, ...others }) => (
+        {visibleBatchActions.map(({ id, labelText, disabled, ...others }) => (
           <TableBatchAction
             key={id}
             onClick={() => onApplyBatchAction(id)}
             tabIndex={shouldShowBatchActions ? 0 : -1}
-            disabled={!shouldShowBatchActions}
+            disabled={!shouldShowBatchActions || disabled}
             {...others}
           >
             {labelText}
           </TableBatchAction>
         ))}
+        {hasVisibleOverflowBatchActions ? (
+          <OverflowMenu
+            data-testid={`${testID || testId}-batch-actions-overflow-menu`}
+            className={`${iotPrefix}--table-overflow-batch-actions`}
+            flipped={langDir === 'ltr'}
+            direction="bottom"
+            onClick={(e) => e.stopPropagation()}
+            renderIcon={OverflowMenuVertical20}
+            tabIndex={shouldShowBatchActions ? 0 : -1}
```

Comment:
Why is this needed?

Reply:
To prevent the button from being tabbed to when the batch action bar is hidden.

Reply:
It's not completely removed from the DOM. It's just hidden. That's why we had to add this fix a few months ago, because people were able to tab to these hidden elements.

Reply:
Ok, this is strange, the batch action bar is really completely removed from the DOM in this story https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-table--with-selection-and-batch-actions (which is where I looked first) but in the playground story it is just hidden...

Reply:
🤔 Hmmm. It looks like that story is causing a full re-render, notice how the bar slides up smoothly from the top of the table header in the playground as items are selected or unselected, but just abruptly disappears in the selection and batch action story.